pagopa/pn-frontend
# manage permissions for operator with groups - pn-6007

## Short description
Now we manage the permissions also for the operator with groups.

## List of changes proposed in this pull request
- Changed isGroupAdmin flag with hasGroup
- Replaced isGroupAdmin flag with hasGroup in all components that used it
- Removed group check inside the DomicileBanner component

## How to test
- PG -> login with an admin -> check that you can view notifications (delegated and not), you can manage mandates (from the PG and to PG) and you can manage contacts
- PG -> login with a group admin -> check that you can only view delegated notifications, you can manage mandates (to PG and only associated to user group) and you cannot manage contacts
- PG -> login with operator -> check that you can view notifications (delegated and not), you cannot manage mandates and you cannot manage contacts
-  PG -> login with group operator -> check that you can only view delegated notifications, you cannot manage mandates and you cannot manage contacts
